src/Resolvers/Microsoft.DotNet.NativeWrapper/EnvironmentProvider.cs

```diff
@@ -72,7 +72,11 @@ public string GetDotnetExeDirectory()
 
             if (string.IsNullOrWhiteSpace(dotnetExe))
             {
+#if NET6_0_OR_GREATER
```

Comment:
Sometimes you use `NET5_0_OR_GREATER`, and here you use `NET6_0_OR_GREATER`. We should be consistent.

Reply:
ProcessId was added in 5.0, ProcessPath in 6.0.